### PR TITLE
➕ add prettier config

### LINKED ISSUE #210: Add Prettier
⭐️ S (situation) ⭐️

You don't need to discuss code changes about formatting: indents, spaces or tabs, etc

⭐️ T (task) ⭐️

The developer will add code changes with declared formatting which is available in Prettier config

⭐️ A (action) ⭐️

Add Prettier with https://prettier.io/

⭐️ R (result) ⭐️

It will autochange files and format them correctly for the project

### PR DESCRIPTION
**Pull request by tyczynski**
PR resolves #210

### CODE REVIEW

**Comment by Beraliv** [comment]
@tyczynski I'm really sorry to make the review too long

I'm looking at it now, will provide the feedback during an hour

File: lib/median/index.js [hunk 1]
```diff
@@ -1,15 +1,14 @@
 import mean from '../mean'
 
-const median = (list) => {
-  var width = 2 - list.length % 2
+export default (list) => {
```
Comment (Beraliv) [inline lib/median/index.js:3]: let's revert the change for now

It will replace empty string with the function name in `SIZES.md`

File: SIZES.md [hunk 2]
```diff
@@ -281,10 +281,10 @@
 | zipWith | 171 B | 292 B | -121 B |
 | --- | --- | --- | --- |
 | minimum | 1 B | 17 B | -16 B |
-| median | 130 B | 364 B | -234 B |
+|  | 130 B | 364 B | -234 B |
```
Comment (Beraliv) [inline SIZES.md:284]: We should fix it here and below in `SIZES_CJS.md`

**Review by Beraliv — CHANGES_REQUESTED** [review]

**Comment by tyczynski** [comment]
@Beraliv 
Thanks for code review. I will make fixes this week :)

**Pull request closed without merging**